dissect/target/plugins/os/unix/esxi/_os.py

```diff
@@ -190,6 +199,53 @@ def configstore(self) -> dict[str, Any]:
     def os(self) -> str:
         return OperatingSystem.ESXI.value
 
+    def _mount_nfs_shares(self) -> None:
+        # Mount NFS shares
+        if not self._configstore:
+            self.target.log.warning("No configstore found")
+            return
+
+        nfs_shares: dict[str, Any] = self._configstore.get("esx", {}).get("storage", {}).get("nfs_v3_datastores", {})
+        if not nfs_shares:
+            self.target.log.info("No NFS shares found in datastore")
```

Comment:
Shouldn't we check if NFS mounting is desired before logging a bunch of things?

Reply:
From discussions with the analysts I gathered they want to be informed when there are nfs stores, even with nfs disabled. (Although strictly speaking, this message states there are *no*  nfs stores :)). 

Maybe debug log level is more appropriate in this case?

Reply:
Nitpicky, but shouldn't that then be a log in `acquire` and not `dissect.target`?

Reply:
Technically this code could also be invoked form target-shell or target-query in combination with the  local loader. 

I do think the warnings should be easier to see when running acquire, will spar a bit with Michaël tomorrow

Reply:
No problemo